Core/include/Acts/Surfaces/DiscSurface.hpp

```diff
@@ -256,7 +256,7 @@ class DiscSurface : public Surface {
   /// @param bcheck The boundary check prescription
   /// @param tolerance the tolerance used for the intersection
   ///
-  ///  <b>mathematical motivation:</b>
+  /// Mathematical motivation:</b>
```

Comment:
opening tag missing now?

```suggestion
  /// <b>Mathematical motivation:</b>
```